js/form.js

```diff
@@ -0,0 +1,61 @@
+import {pristine} from './libs/pristin-init.js';
+import {form, typeField, priceField, typesMinPrice, timeFieldset, range} from './const.js';
+import {blockSubmitButton, setValue, isSuccessSubmit, isErrorSubmit, resetMarker} from './utils.js';
+import {sendData} from './api.js';
+
+const onFormSubmit = (evt) => {
+  evt.preventDefault();
+  const isValid = pristine.validate();
+
+  if (isValid) {
+
+    blockSubmitButton();
+
+    sendData(
+
+      () => {
+        isSuccessSubmit();
```

Comment:
```suggestion
        onSuccessSubmit();
```
это всё - ф-и обработчики - а для слова `обработчик` ты выбрала `on`. Это же просто перевод слова handler - обработчик. Да, в js принято два названия для этого.

Reply:
https://up.htmlacademy.ru/javascript/25/criteries#d4 всё этот критерий

Reply:
ну и тут тебе надо просто передать название ф-и обработчика, которая в sendData будет использоваться как колбек. Тебе не надо передавать саму ф-ю